woocommerce/woocommerce
# Load `docker-compose.override.yml` if it exists Fix #36477

### Changes proposed in this Pull Request:

During E2E testings, optionally load a `docker-compose.override.yml` from the project root.

Closes #36477 .


### How to test the changes in this Pull Request:



Similar to the description in https://github.com/woocommerce/woocommerce/issues/36477

1. Create `docker-compose.override.yml` in the project root with the contents:
```
services:
  php:
    image: php:8.1
```

2. Run E2E tests
3. See in Docker that PHP 8.1 is being using by the PHP image in the container

### Other information:


### Questions

* Should it be named `docker-compose.override.yml` as the Docker documentation suggests, or `docker-compose.override.yaml` to match the WooCommerce convention for `docker-compose.yaml`? (or check for both?)
* Is `../../../docker-compose.override.yml` the correct number of parent directories in all cases? (It is correct when using E2E Boilerplate)

### FOR PR REVIEWER ONLY:
